manual/manual/tutorials/coreexamples.etex

```diff
@@ -54,11 +54,23 @@ fib 10;;
 \pdfsection{Data types}
 
 In addition to integers and floating-point numbers, OCaml offers the
-usual basic data types: booleans, characters, and immutable character strings.
+usual basic data types: booleans,
```

Comment:
It looks like the enumeration (and the sentence as a consequence)
is not properly terminated.

Reply:
@xclerc Explain? I'm not sure I get your point.

Reply:
Well, if I am correct, the paragraph is "*In addition to integers and floating-point
numbers, OCaml offers the usual basic data types: booleans,*", then followed by
an example. It seems incorrect to end the paragraph with a comma (and having
an enumeration with only one item is unexpected).

Reply:
It isn't the end. There are two other things in the enumeration, characters and strings. What's happened is that examples have been interspersed.

Reply:
Oh, I see -- sorry for the noise.

Reply:
Nevertheless, code examples interleaved with the text (and integrated with the punctuation) might be unusual. Should I switch to a (bullet point) list to improve readability?

Reply:
I haven't actually formatted the manual with your changes (I read the LaTex directly) so I'm not sure. Why don't you look at it both ways and pick one?

Reply:
After taking a look at both options, I picked the itemized list, since I found that the indentation gives better visual cues to the reader.

Reply:
Seems good to me. Is this a candidate for cherry-picking to 4.07?